### ISSUE TITLE
HP Elitebook 840 G1 - Microphone (Fn+F8) LED and mute not working

### ISSUE THREAD

**Issue by TheSimpleMachine** [systemd/systemd#15774]
On the HP Elitebook 840 G1 the "Fn+F8" combination does not disable to microphone mute LED and also does not actually mute the Microphone. This has been a consistent issue since 18.04 up to 20.04.

I reported the bug and a solution that worked:
https://bugzilla.kernel.org/show_bug.cgi?id=207487
https://answers.launchpad.net/ubuntu/+source/systemd/+question/690306

Steps to resolve the problem replicated below:

Open a terminal window and issue the command
gedit ~/61-hp-mic-mute-hotkey.hwdb

Add the following two lines in the file that opens. 

evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*EliteBook*:pvr*
 KEYBOARD_KEY_81=f20                          # Fn+F8 on Elitebook, map to F20

Save the file and close the editor.

Next, issue the following commands:

sudo cp ~/61-hp-mic-mute-hotkey.hwdb /lib/udev/hwdb.d/
sudo systemd-hwdb update
sudo udevadm trigger
rm ~/61-hp-mic-mute-hotkey.hwdb

At this point the Fn+F8 combo key functions to turn on/off the microphone and the associated led light.
 
Hoping this can be turned into an out of the box solution in future linux versions. This would resolve the issue across various HP laptops. Especially non technical users like myself.
https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1705586
https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1334968
https://wiki.archlinux.org/index.php/HP_EliteBook_840_G1#Enable_the_microphone_muting_key

**Comment by poettering** [comment]
Please submit a PR for this!

**Comment by TheSimpleMachine** [comment]
> Please submit a PR for this!

What is a PR? I am new to ubuntu and github and trying to learn the ropes.

**Comment by TheSimpleMachine** [comment]
Disclaimer: I have searched around trying to understand how to create the pull request. Being a non technical person and fairly new to this I am lost.

I tried to follow these two links for direction.
https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
